Projects/UOContent/Mobiles/Monsters/Humanoid/Magic/EvilMage.cs

```diff
@@ -39,8 +40,17 @@ public EvilMage() : base(AIType.AI_Mage)
 
             VirtualArmor = 16;
             PackReg(6);
-            PackItem(new Robe(Utility.RandomNeutralHue()));
-            PackItem(new Sandals());
+            
+            if (Core.UOR)
+            {
+                PackItem(new Robe(Utility.RandomNeutralHue()));
+                PackItem(new Sandals());
+            }
+            else
+            {
+                EquipItem(new Robe(Utility.RandomNeutralHue()));
+                EquipItem(new Sandals());
+            }
```

Comment:
Let's just equip it.

```suggestion
            EquipItem(new Robe(Utility.RandomNeutralHue()));
            EquipItem(new Sandals());
```